Doc/library/subprocess.rst

```diff
@@ -1149,6 +1149,11 @@ calls these functions.
    code was zero then return, otherwise raise :exc:`CalledProcessError`. The
    :exc:`CalledProcessError` object will have the return code in the
    :attr:`~CalledProcessError.returncode` attribute.
+   .. note::
+
+      If .. function:: `check_call` was run unsuccessfully - it can raise only
+      :exc:`CalledProcessError`, however there might be other exceptions if the
+      command could not run at all.
```

Comment:
Thank you for picking this up.

I think this wording is not exactly correct - the "check_call" function runs in any case. The two failures we are distinguishing are (1) check_call was unable to start the process you tried to run and (2) It did manage to launch your program, but your program failed (returned a non-zero error code).

In the first case, you get the error that occurred, in the latter case check_call raises a CalledProcessError that wraps the actual return code.

Reply:
Thank you for your explanation! I've updated the request